### PR TITLE
Account selection sheet design review comments

### LINKED ISSUE #19197: Communities - Request to Join drawer issues
# Design Feedback 2.27

## [Communities] Request to join drawer screen issues
Figma frame of the Review [here](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj/Release-Reviews?type=design&node-id=4%3A111403&mode=design&t=IwKFMlZGTBic5nwK-1)

- **Upper margin** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=103:77903&mode=design#714520830)] between the top of the drawer and the close button
- **Account list preview** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=4:111403&mode=design#726026933)] size is smaller than design
- **Community rules** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=103:77903&mode=design#714528125)] list has a random spacing after the 2nd rule

Screenshot 2024-03-12 alle 14 10 24

### Reproduction

1. Join a community

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS
- Devices in the screenshots: iPhone 15 Pro Max

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #19197

### Summary
image

Increases image dimensions and adjust top spacing between close button and top of the sheet

status: ready

### CODE REVIEW

File: src/status_im/contexts/communities/actions/accounts_selection/style.cljs [hunk 1]
```diff
@@ -5,7 +5,8 @@
 (def screen-horizontal-padding 20)
 
 (def container
-  {:flex 1})
+  {:flex       1
+   :margin-top -12})
```
Comment (smohamedjavid) [inline src/status_im/contexts/communities/actions/accounts_selection/style.cljs:9]: Any reason why we set `-12`?

EDIT: If this is applied to override the padding/margin top of the bottom sheet, can we add it as a prop in the bottom sheet?

Comment (ibrkhalil) [reply src/status_im/contexts/communities/actions/accounts_selection/style.cljs:9]: Sure, Will do!

Comment (ibrkhalil) [reply src/status_im/contexts/communities/actions/accounts_selection/style.cljs:9]: Hey Mohamed, Do you think it's better for the bottom-sheet changes to be a followup?
For the sake of easing QA review and minimizing this PR's scope.

Comment (smohamedjavid) [reply src/status_im/contexts/communities/actions/accounts_selection/style.cljs:9]: Okay 👍

**Review by smohamedjavid — APPROVED** [review]

**Comment by mariia-skrypnyk** [comment]
Hi @ibrkhalil !

Should this PR **skip** manual qa or **not**?
Please, add needed label 🙌

**Comment by ibrkhalil** [comment]
> Hi @ibrkhalil !
> 
> Should this PR **skip** manual qa or **not**? Please, add needed label 🙌

Since it's only a visual change, Then yes it's a good idea
@Francesca-G Need a design review here please.

**Comment by mariia-skrypnyk** [comment]
E2E failed tests are not related.
@ibrkhalil thanks for comment regarding testing label.
In this case PR needs to be moved to Design Review column.

**Review by Francesca-G — APPROVED** [review]
Looks good to me ✨

**Pull request merged**